sof-host-defconfig

```diff
@@ -0,0 +1,10 @@
+CONFIG_TASKSTATS=n
+CONFIG_TASK_DELAY_ACCT=n
+CONFIG_KVM=m
+CONFIG_KVM_INTEL=m
+CONFIG_VHOST_NET=m
+CONFIG_VHOST_SOF=y
+CONFIG_BRIDGE=m
+CONFIG_TUN=m
+CONFIG_SND_SOC_INTEL_SOF_PCM512x_MACH=m
```

Comment:
why is this here? this should be dependent on the mach-driver-defconfig file, this is unrelated to virtio IMHO